\documentclass[onecolumn, compsoc,12pt]{IEEEtran}
\usepackage{enumitem}
\input{preamble.tex} 
%\usepackage{cite}
\usepackage{textcomp}
\usepackage{colortbl}
\usepackage{subfigure}
\usepackage{array}
\usepackage{courier}
\usepackage{setspace} 
\usepackage{wrapfig} 
\usepackage{calligra}
\usepackage{ulem}
\usepackage{multirow}
\renewcommand{\IEEEbibitemsep}{20pt plus 2pt}
\makeatletter
\IEEEtriggercmd{\reset@font\normalfont\fontsize{11}{14}\selectfont}
\makeatother
\IEEEtriggeratref{1}
\newlength{\bibitemsep}\setlength{\bibitemsep}{.2\baselineskip plus .05\baselineskip minus .05\baselineskip}
\newlength{\bibparskip}\setlength{\bibparskip}{0pt}
\let\oldthebibliography\thebibliography
\renewcommand\thebibliography[1]{%
  \oldthebibliography{#1}%
  \setlength{\parskip}{\bibitemsep}%
  \setlength{\itemsep}{\bibparskip}%
}
\setlength{\bibitemsep}{.3\baselineskip plus .05\baselineskip minus .05\baselineskip}

\usetikzlibrary{chains,backgrounds}
\usetikzlibrary{intersections}
%\usepackage[super]{cite} 
%\makeatletter \renewcommand{\@citess}[1]{\raisebox{1pt}{\textsuperscript{[#1]}}} \makeatother
\usepackage{xstring}
\usepackage{wasysym}
\usepackage[misc]{ifsym}
\renewcommand{\thesectiondis}{\arabic{section}.}
\renewcommand{\thesubsectiondis}{\Alph{subsection}.}

\makeatletter
\renewcommand\section{\@startsection {section}{1}{\z@}%
                                   {-1pt \@plus -30ex \@minus 20ex}%
                                   {.1pt}%
                                   {\large\bfseries\scshape}}
\renewcommand\subsection{\@startsection {subsection}{2}{\z@}%
                                   {0ex \@plus -1.75ex \@minus -1.2ex}%
                                   {0ex \@plus.0ex}%
                                   {\fontsize{11}{11}\selectfont\bfseries\sffamily\color{black}}}
\renewcommand\subsubsection{\@startsection {section}{1}{\z@}%
                                   {-1.5ex \@plus -.5ex \@minus -.2ex}%
                                   {0.0ex \@plus.5ex}%
                                   {\fontsize{9}{9}\selectfont\bfseries\sffamily\color{Red4}}}
\renewcommand\paragraph{\@startsection {section}{1}{\z@}%
                                   {-.1ex \@plus -.5ex \@minus -.2ex}%
                                   {0.0ex \@plus.5ex}%
                                   {\fontsize{11}{10}\selectfont\bfseries\itshape\sffamily\color{black}}}
\makeatother
 
                          
\makeatletter
\pgfdeclareradialshading[tikz@ball]{ball}{\pgfqpoint{-10bp}{10bp}}{%
 color(0bp)=(tikz@ball!30!white);
 color(9bp)=(tikz@ball!75!white);
 color(18bp)=(tikz@ball!90!black);
 color(25bp)=(tikz@ball!70!black);
 color(50bp)=(black)}
\makeatother
\newcommand{\tball}{${\color{CadetBlue3}\Large\boldsymbol{\blacksquare}}$}
\renewcommand{\baselinestretch}{.96}
\newcommand{\VSP}{\vspace{-2pt}}
\renewcommand{\captionN}[1]{\caption{\color{CadetBlue4!80!black} \sffamily \fontsize{9}{10}\selectfont #1  }}
\tikzexternaldisable 
\parskip=3pt
\parindent=0pt
\newcommand{\Mark}[1]{\textsuperscript{#1}}
\lhead{}
\pagestyle{fancy}
\def\COLA{black}
%###################################
\cfoot{\bf\sffamily \scriptsize \color{Maroon!50} I. Chattopadhyay, Department of Medicine, University of Chicago}
\cfoot{}
\rhead{}
%\rhead{\bf\sffamily \scriptsize \color{DodgerBlue4!50} DARPA Young Faculty Award 2017}
%\rhead{\scriptsize\bf\sffamily \href{zed.UChicago.edu}{zed.UChicago.edu}}
\rfoot{\raisebox{.2in}{\scriptsize\bf\sffamily\thepage}}
\newcommand{\partxt}{\bf\sffamily\itshape}
% ############################################################
\draftQtrue

\newif\ifFIGS
\FIGStrue
\newif\iftikzX
\tikzXtrue
\tikzXfalse

\newcommand\guline{\bgroup\markoverwith
{\textcolor{black!30}{\rule[-0.45ex]{2pt}{0.4pt}}}\ULon}
\newcommand\hilit[1]{\textcolor{Red1}{#1}}
\newcommand\hilitx[1]{\guline{#1}}
%############################################################
\addtolength{\voffset}{.1in}
\addtolength{\textwidth}{-.085in}
\addtolength{\hoffset}{.0425in}
\def\PROG{Mallinckrodt\xspace}
\def\ZERO{ACoR\xspace}
\def\COLWA{\XCOLA!40}
\def\COLWB{\XCOLD!20}
\def\COLWC{\XCOLA!40}
\def\COLWD{\XCOLD!20}
\def\COLWE{\XCOLA!40}
\def\COLWF{\XCOLD!20}
% ############################################################
\def\treatment{positive\xspace}
\input{customdef}

\def\V{\mathds{V}}
\def\hcov{SARS-CoV-2\xspace}
\def\RATG13{RaTG13\xspace}
\def\Appendix{Appendix}
\def\qnet{Qnet\xspace}
\def\cov{COVID-19\xspace}
\def\infl{Influenza A\xspace}
\def\PATH{../pnas/}
%###################################
\def\MONO{mono}
%\def\MONO{}
\begin{document} 

\vspace{20pt}



\clearpage
\setcounter{page}{1}


$\phantom{x}$
\vspace{-35pt}  

\section*{Predicting Future Mutations for  Escape-resistant Vaccines}
The continuing mutation of \cov (delta, lambda, omicron) during \cov pandemic  has shown the need for a new type of vaccine designs - one that is as dynamic and nimble as the virus it plans to protect against. Periodic reformulations similar to  seasonal  flu vaccines,  is  problematic for \cov given its current rapid rate of mutation coupled with a high transmissibility, infectious asymptomatic patients, vaccine hesitancy, and potentially high mortality. This situation is not unique: emergent viruses experience diverse  selection pressures fostering  adaptations via new mutations. The current state of knowledge has no reliable tools to preempt such viruses: we do not know when or how new mutants will arise, and how to protect against them (Hum. vacc. \& immunotherapeutics 16:286-294,'20),(Global Catas. Bio. Risks  75--83,10.1007/82\_2019\_179).%~\cite{gou2020systematic}\cite{fair2019viral}.
Thus, there is a need of revolutionary conceptual  breakthroughs to predict how a viral strain  mutates in the wild under realistic selection, allowing for the design and testing of vaccines before the emergence event.

\paragraph*{Unique Aspects} To achieve this goal, we formulated the methodological foundations for a deep understanding of the evolutionary dynamics  in  the sequence/strain space. Our overarching vision, backed by pilot studies over the past year with limited intramural  funding from the UChicago Big Ideas  incubator, is to  computationally interrogate  evolutionary patterns driving  the current  pandemic and beyond. Since each  strain is but a single point in a  $\approx  \times10^4$ dimensional space (\hcov genome  $\approx 3 \times 10^4$ bases), we can never comprehensively explore the  state space. But we don't need to. We reduce the number of combinations by accounting for only those that occur along evolutionary trajectories â€“ making calculation possible on high-performance computing clusters.

We have to-date predicted new mutations on the \hcov spike protein, and shown in in-vitro experiments that these  predicted variants express correctly, are functional (bind to the human ACE2 receptor), and some are more resistant to antibody binding assays compared to the wild type. \uline{Using data from early days of the pandemic, we could preempt mutations that eventually arose in the delta variant.} Testing the idea along a longer time-frame, we applied the same concept to Influenza drift. Here, this approach consistently out-performed WHO/CDC predictions for vaccine components with respect to how far removed the predictions were  from the dominant strain in the future season (Medarxiv, 10.1101/2020.07.17.20156364).%~\cite{Li2020.07.17.20156364}.

\paragraph*{Personnel} Thus, via a cross-disciplinary collaboration  between Prof. Ishanu Chattopadhyay (mathematical modeling, information theory, machine learning) and Prof. Aaron Esser-Kahn (immunology, vaccine science), we envision a radically different approach to escape-resistant vaccine design. Beyond  predicting likely future mutations in circulating strains, the goal of this proposal will be to build a platform technology which can be developed/tested toward (1) predicting mutations within a single individual as a potential source of novel variant emergence, and (2) develop a rank-ordering of  sampled strains in animal reservoirs by  risk of emergence (a capability well-beyond the state of the art). Such methods would form the nucleus of a burgeoning field of precision  interventions  in the animal reservoir to preemptively neutralize threats,  \textit{before the first human infection}.

\paragraph*{Justifying Keck Support} Our vision   entails  risks;  we are challenging a prevailing dogma, that future mutations, and variants, of a pathogen are intrinsically random and hence unpredictable.  We have sufficient evidence to the contrary, and need Keck's support to validate our tool in a well-vetted test/design/test loop  ultimately fostering a paradigm shift in  how we combat pandemics in future. While we have been turned down recently by NIH (FOA: AI21-035, Application id: 1 R21 AI169352-01), this study can fundamentally change the game, with high future interest from stakeholders, including NIH and Biological Tech. office at DARPA.

\paragraph*{Budget, Timeline} Conducted  over a period of four years costing  1.5M USD, supporting study personnel (PI time + Postdoctoral time $\approx 60\%$, computational costs ($\approx 10\%$) and experimental costs ($\approx 30\%$), with some allocation for travel, and publication funding. 
  


\clearpage
\limitpages{2}

\setcounter{page}{1}

\section*{Project Description}
\paragraph*{Overview} The \cov pandemic, despite multiple vaccines, continues to be an ongoing challenge as new variants and potential escape mutants emerge. The current practice has no tools to predict, let alone preempt such emergence: we do not know when  new mutants  will arise, and how these mutants will differ in terms of pathogenicity, transmissibility and resistance to current vaccines.  A key conceptual barrier  is the missing ability to numerically estimate the likelihood of specific future mutations. Currently this likelihood is  equated to sequence similarity, which is measured by how many mutations it takes to change one strain to another (the edit distance). In reality, the odds of one sequence mutating to another is a function of not just how many mutations they are apart at the beginning, but also how specific mutations incrementally affect fitness. Ignoring the constraints needed to conserve function makes any assessment of the mutation likelihood suspect. In this study we plan to computationally learn these complex and hitherto unknown evolutionary constraints from large sequence databases, enabling us to the chart   trajectories of wild pathogens at scale. We propose to experimentally validate our approach  in binding and neutralization assays,  allowing us to leverage  sequence and structural  annotation databases, to predict when and how new strains are expected to appear, along with their  impact on pathogenicity, and vaccine escape. 
\paragraph*{Relevant Efforts}  The  Big Ideas Generator (BIG) program at the University of Chicago  has funded our initial work, with  substantial interest going forward with staff and utility support.
\paragraph*{Peer Groups} Very recently, two articles investigated predicting pathogenicity  from genomic sequences (Mollentze (PLoS biology 19: e3001390, '21)), and identifying current mutations which might dominate in future (Maher $\etal$ (Science trans. med. eabk3445,'21)). While these questions overlap with our framework,  our approach is distinct, and vastly more ambitious both intellectually and in scope. Mollentze  uses  classical  sequence similarity to  human housekeeping genes hoping to identify viruses  evading the human immune system, with limited   performance (tagging incorrectly all SARS-related coronaviruses as pathogenic). And, Maher assumes mutations are mutually independent ignoring crucial epistatic and compensatory effects (Curr. opinion in struct. bio. 50: 18--25,'18), combining  manually curated \hcov-specific putative features via machine learning. Importantly, these approaches  % treat a complete  strain (or the RBD) as the target object of prediction or modeling -
 aim to predict point mutations, not addressing the next-level challenge of tracking dependent and compensatory mutations throughout a complete strain. Such ultra-high-dimensional sequence space predictions  lie well beyond reach of our peers. While useful in hindsight analysis, these approaches cannot yet predict a new strain with an estimated probability, or predict whether it will pose a threat.  Our method goes deeper into the fundamental principles underlying viral evolution, using information from each sequence to make strain-specific predictions. Thus, with sufficient data we can track any species,  its future mutations and emergence events. %This is crucial -- the next pandemic may not be a CoV derivative.

\paragraph*{Goals \& Methodology}
We computationally infer a collection of cross-dependent predictors (the Q-net) that maximally extracts dependency information between mutations \& motifs. We can preempt complete strains that have never been seen before, but nevertheless represent a valid genomic sequence. Our framework is general. With no manually curated features for  individual viral species, our \textit{sequence only} model lends robust scalability. 
Our goals (Fig.~1): \begin{enumerate} 
[label=$\square$, leftmargin=0pt,
labelindent=0em, topsep=0.1em, labelsep=*, itemsep=.25em,itemindent=1em]

\item \textbf{Aim 1: Validate  a meaningful comparison (q-distance) for genomic sequences. (12 mo, Y1)}  Combining novel machine learning, and  information theory, we will  characterize  mutation patterns  from large sequence databases (Nuc Acids Res 45:D482--D490, 16) that constrain evolutionary trajectories and reveal selection pressures,  to inform a biologically meaningful species-specific adaptive metric of sequence similarity. Using \hcov \& \infl as model organisms, we will validate the q-distance using past trajectories  of dominant strains, showing closer sequences in this metric are more predictive of phenotype than edit distance.
  %
\item \textbf{Aim 2: Develop+validate  algorithm preempting future variants (24 mo, Y1-Y2) } With  tractable function-aware sampling (q-sampling) of the neighborhood of an observed strain in ultra-high-dimensional  possibility space, we will preempt: 1) \uline{future likely mutations} 2) probability of spontaneous jump  via specific mutations, and 3)  likely variants arising within specified time-frames in the wild. Validate  that predicted mutations/strains  are biologically plausible,  expressing functional proteins, both  in silico and in laboratory assays, piloted with the spike protein for \hcov and Hemagglutinin (HA) for Influenza A. In each case, we will predict $10^3$ in silico sequences of each protein, validate  folding  using standard software (Nat Struct \& Mol Bio 28:869--870, '21), then screen top-candidates for in-vitro assays.
  %
\item \textbf{Aim 3. Preempt and characterize escape variants. (36 mo, Y2-Y4) } Preempt escape variants, via characterizing future mutations that  \uline{evade standard antibody neutralization assays}, and thus are candidate escape mutants for  \hcov and \infl. Taking the 1,000 pseudo-virus expressing proteins , we will down-select for proteins that (1) bind either ACE-2 or Sialic Acids (est 100), and then (2) escape the binding of panels of sera from convalescent patients and vaccine recipients (estimate 10-50). These $\approx$10-50 protein sequences will then be encoded as model mRNA vaccines and their antibody responses evaluated. The success metric here is to show that our tools significantly reduce escape odds. This characterization will also feed-back to fine-tune q-sampling to directly predict, escape mutants with high probability.
  % 
\item \textbf{Aim 4. Define the emergence edge identifying animal strains  poised to emerge into humans with high transmissibility/pathogenicity. (24 mo, Y3-Y4)}  Piloted with emerging \infl strains, compare  predictions \uline{against CDC-developed Influenza Risk Assessment Tool (IRAT) scores}  for flu variants, and  characterize all Influenza A sequences (and predicted variants)  in public databases.  If validated, we will vastly accelerate risk assessment, cutting down the time required for individual strains from weeks/months to $< 1$sec.
\end{enumerate}
%
\begin{wrapfigure}[17]{l}{4.7in}
\vspace{-25pt}

  \includegraphics[width=4.8in]{Figures/fig0\MONO}
  
    \vspace{-18pt}

  \captionN{Conceptual framework and outcomes: General approach to scalably learn mutational  dependencies to predict future escape variants and proactive surveillance }\label{fig1}
  \vspace{5pt}
  
\end{wrapfigure}\paragraph*{Impact}  Adhoc quantification of genomic similarity (Bioinformatics 14:817--818, '98) (Elife 3: e03568, '14) is not meaningful -- a smaller edit distance between two strains  does not imply  a feasible wild trajectory  from one to the other.  Our  algorithms are a first to learn the \textbf{appropriate metric of comparison}, without assuming any model of DNA/RNA substitution, or a genealogical tree a priori, and are  designed to be aware of the impact of the  host environment and background epidemiology. This study, if successful, will have profound impact on  biosurveillance strategies, and what we do with the products of such efforts. With  risk-wise rank-ordering of  newly collected strains, we can  1) better judge pandemic risks, 2) quantify the odds of a particular strain spilling to humans, and 3) estimate its potential to lead to a global pandemic. And for strains already circulating in the human population, we can potentially  4) preempt variants, 5) their timeline of emergence,  6) their odds of escaping current vaccines, and ultimately 7) design escape-resistant vaccines.

\textbf{Fundraising:} To date $\approx$100K allocated (BIG funding + PI development fund). We have pending proposals at NSF (PIPP, 1M USD, summer '22) and NIH (R21, 400K, Fall '22). 

\clearpage

\input{pagecontrol.tex}
\limitpages{8}

\setcounter{page}{1}
%
\section*{Knowledgeable Experts in the field}
1. Peter Hraber\\
Theoretical Biology \& Biophysics Group, T-6\\
Theoretical Division\\
Los Alamos National Laboratory\\
PO Box 1663, MS K710\\
Los Alamos, NM 87545\\
phone: 505 665 7491\\
email: phraber@lanl.gov\\

Dr. Hraber is an expert in  theoretical biology and biophysics,  focusing in  computational immunology, evolution, and statistical genetics, and is well-suited to evaluate the interplay of mathematical modeling, evolutionary dynamics and immunological aspects of the proposed project.
\vskip 1em
\hrule

2. Patrick Wilson\\
Assistant Professor\\
Drukier Institute for Children's Health\\
Weill Cornell Medicine\\
1300 York Avenue\\
New York, NY 10065\\
phone: 212 746 4111\\
email: pcw4001@med.cornell.edu\\

Prof. Wilson is an immunologist with extensive experience in characterization of human immune responses, with definitive work in influenza vaccine designs and B cell biology.
\vskip 1em
\hrule

3. Balaji Manicassamy\\
Associate Professor of Microbiology and Immunology\\
Iowa State University\\
3-430 Bowen Science Building\\
51 Newton Rd\\
Iowa City, IA 52242\\
phone: 319 335 7590\\
email:  balaji-manicassamy@uiowa.edu\\

Prof. Manicassamy is an expert in influenza viruses and respiratory pathogens, with extensive experience in reverse genetics and pathogenesis.
\vskip 1em
\hrule

4. Geoffrey Lynn \\ 
Senior Vice President, Synthetic Immunotherapies at Vaccitech\\
1812 Ashland Ave\\ Baltimore, MD 21205\\
Bethesda, Maryland, United States\\
%phone: \\
email: Geoffrey.Lynn@vaccitech.us\\

Dr. Lynn is an expert in synthetic chemistry and cellular immunology with research interest in precision immunotherapies for complex diseases. 
\vskip 1em
\hrule

5. Danny Altmann \\
5S5C Hammersmith Hospital\\
Hammersmith Campus\\
72 Du Cane Rd, London W12 0HS, United Kingdom\\
phone: +44 (0)20 3313 8212\\
email:  d.altmann@imperial.ac.uk\\

Prof. Altmann is an a well-known immunologist with research interest in the immunology of  infectious disease including severe bacterial infections.
\vskip 1em
\hrule
\clearpage

\normalem 
 
%\bibliographystyle{abbrvnat}
%\bibliography{keck,qnet}

 \begin{thebibliography}{11}
\providecommand{\natexlab}[1]{#1}
\providecommand{\url}[1]{\texttt{#1}}
\expandafter\ifx\csname urlstyle\endcsname\relax
  \providecommand{\doi}[1]{doi: #1}\else
  \providecommand{\doi}{doi: \begingroup \urlstyle{rm}\Url}\fi

\bibitem[Bagdonas et~al.(2021)Bagdonas, Fogarty, Fadda, and
  Agirre]{bagdonas2021case}
H.~Bagdonas, C.~A. Fogarty, E.~Fadda, and J.~Agirre.
\newblock The case for post-predictional modifications in the alphafold protein
  structure database.
\newblock \emph{Nature Structural \& Molecular Biology}, 28\penalty0
  (11):\penalty0 869--870, 2021.

\bibitem[CDC()]{Influenz24:online}
CDC.
\newblock Influenza risk assessment tool (IRAT)
\newblock
\url{https://www.cdc.gov/flu/pandemic-resources/national-strategy/risk-assessment.htm}.
\newblock Dept. of HHS Report (no DOI).
\newblock
  \url{https://www.cdc.gov/flu/pandemic-resources/pdf/CDC-IRAT-Virus-Report.pdf}.
\newblock (Accessed on 07/02/2021).


\bibitem[Fair and Fair(2019)]{fair2019viral}
J.~Fair and J.~Fair.
\newblock \emph{Viral Forecasting, Pathogen Cataloging, and Disease Ecosystem
  Mapping: Measuring Returns on Investments}, pages 75--83.
\newblock Springer International Publishing, Cham, 2019.
\newblock ISBN 978-3-030-36311-6.
\newblock \doi{10.1007/82_2019_179}.

\bibitem[Gou et~al.(2020)Gou, Wu, Shi, Zhang, and Huang]{gou2020systematic}
X.~Gou, X.~Wu, Y.~Shi, K.~Zhang, and J.~Huang.
\newblock A systematic review and meta-analysis of cross-reactivity of
  antibodies induced by h7 influenza vaccine.
\newblock \emph{Human vaccines \& immunotherapeutics}, 16\penalty0
  (2):\penalty0 286--294, 2020.

\bibitem[Hatcher et~al.(2016)Hatcher, Zhdanov, Bao, Blinkova, Nawrocki,
  Ostapchuck, SchÃ¤ffer, and Brister]{Hatcher_2016}
E.~L. Hatcher, S.~A. Zhdanov, Y.~Bao, O.~Blinkova, E.~P. Nawrocki,
  Y.~Ostapchuck, A.~A. SchÃ¤ffer, and J.~R. Brister.
\newblock Virus variation resource {\textendash} improved response to emergent
  viral outbreaks.
\newblock \emph{Nucleic Acids Research}, 45\penalty0 (D1):\penalty0 D482--D490,
  nov 2016.
\newblock \doi{10.1093/nar/gkw1065}.

\bibitem[Li et~al.(2020)Li, Li, and Chattopadhyay]{Li2020.07.17.20156364}
J.~Li, T.~Li, and I.~Chattopadhyay.
\newblock Preparing for the next pandemic: Learning wild mutational patterns at
  scale for analyzing sequence divergence in novel pathogens.
\newblock \emph{medRxiv}, 2020.
\newblock \doi{10.1101/2020.07.17.20156364}.

\bibitem[Maher et~al.(2021)Maher, Bartha, Weaver, Di~Iulio, Ferri, Soriaga,
  Lempp, Hie, Bryson, Berger, et~al.]{maher2021predicting}
M.~C. Maher, I.~Bartha, S.~Weaver, J.~Di~Iulio, E.~Ferri, L.~Soriaga, F.~A.
  Lempp, B.~L. Hie, B.~Bryson, B.~Berger, et~al.
\newblock Predicting the mutational drivers of future sars-cov-2 variants of
  concern.
\newblock \emph{Science translational medicine}, page eabk3445, 2021.

\bibitem[Mollentze et~al.(2021)Mollentze, Babayan, and
  Streicker]{mollentze2021identifying}
N.~Mollentze, S.~A. Babayan, and D.~G. Streicker.
\newblock Identifying and prioritizing potential human-infecting viruses from
  their genome sequences.
\newblock \emph{PLoS biology}, 19\penalty0 (9):\penalty0 e3001390, 2021.

\bibitem[Neher et~al.(2014)Neher, Russell, and Shraiman]{neher2014predicting}
R.~A. Neher, C.~A. Russell, and B.~I. Shraiman.
\newblock Predicting evolution from the shape of genealogical trees.
\newblock \emph{Elife}, 3:\penalty0 e03568, 2014.

\bibitem[Posada and Crandall(1998)]{posada1998modeltest}
D.~Posada and K.~A. Crandall.
\newblock Modeltest: testing the model of dna substitution.
\newblock \emph{Bioinformatics (Oxford, England)}, 14\penalty0 (9):\penalty0
  817--818, 1998.

\bibitem[Storz(2018)]{storz2018compensatory}
J.~F. Storz.
\newblock Compensatory mutations and epistasis for protein function.
\newblock \emph{Current opinion in structural biology}, 50:\penalty0 18--25,
  2018.

\end{thebibliography}


\end{document}

